Dockerfile

```diff
@@ -1,51 +1,68 @@
 # syntax=docker/dockerfile:1
 
-ARG GO_VERSION="1.22"
-ARG ALPINE_VERSION_BUILDER="3.18"
-ARG ALPINE_VERSION_RUNNER="3.19"
```

Comment:
I feel like we're risking reintroducing the SIGABRT wasmvm issue we saw on v8 with these changes to golang and alpine.

We might want to consider spawning a temp sidechain on current testnet data, and observe them over several hours to make sure the nodes don't panicloop on us.

Reply:
resolving, went a full weekend running the build without a SIGABRT